test/uk/gov/hmrc/cdsreimbursementclaim/models/Generators.scala

```diff
@@ -21,16 +21,20 @@ import org.joda.time.DateTime
 import org.scalacheck.ScalacheckShapeless._
 import org.scalacheck.{Arbitrary, Gen}
 import reactivemongo.bson.BSONObjectID
-import uk.gov.hmrc.cdsreimbursementclaim.models.declaration.request.{DeclarationRequest, OverpaymentDeclarationDisplayRequest, RequestCommon, RequestDetail}
-import uk.gov.hmrc.cdsreimbursementclaim.models.declaration.response._
-import uk.gov.hmrc.cdsreimbursementclaim.models.declaration.{Declaration, MaskedBankDetails}
 import uk.gov.hmrc.cdsreimbursementclaim.models.upscan.UpscanCallBack.{UploadDetails, UpscanSuccess}
 import uk.gov.hmrc.cdsreimbursementclaim.models.upscan.{UploadReference, UpscanUpload}
-
 import java.time.{Instant, LocalDate, LocalDateTime, ZoneId}
+
 import scala.reflect.{ClassTag, classTag}
 
-object Generators extends GenUtils with UpscanGen with DeclarationGen {
+object GenerateUpscan extends Generators with UpscanGen
+object GenerateDeclaration extends Generators with DeclarationGen
+object GenerateSubmitClaim extends Generators with SubmitClaimGen
+object GenerateFrontendSubmitClaim extends Generators with FrontendSumbitClaimGen
+object GenerateUploadFiles extends Generators with UploadFileGen
+object GenerateWorkItem extends Generators with WorkItemGen
+
```

Comment:
what was wrong with following the existing pattern to create new generators using traits?

Reply:
Name collisions